configs/bundle_config.yaml

```diff
@@ -159,6 +159,16 @@ databundles:
     output:
     - data/natura.tiff
 
+  bundle_cutout_sea:
+    countries: [ASEAN]
+    category: cutouts
+    destination: "cutouts"
+    urls:
+      zenodo:
+      gdrive:
+    output:
+    - [cutouts/sear-2013-era5.nc]
```

Comment:
Hello @Tomkourou !
We just discussed today that we better have the same naming for all cutouts to ease the use by first-time users.

All cutouts shall be named cutouts/cutout-2013-era5, so with format {cutout}-{year}-era5.nc

This because for a first user, the only think to do is select the target country/(ies) and everything follows.
Otherwise, you need to also change the electricity section to match the cutout.

As a comment, when you zip the file, please, use the zip_folder.py function for compatibility purposes

Reply:
I will change the .nc file name to adhere to this format @davide-f. Do you want me to rename the other cutouts as well?

Reply:
No, please, focus on sear only.
Changing the other bundle configs imply to revise the files in zenodo and gdrive as well.
That may be done in another PR.
If interested, that would be a nice task as well. I'll open an issue maybe

Reply:
Here it is :) https://github.com/pypsa-meets-earth/pypsa-earth/issues/758